src/main/resources/db/migration/V2__work_offer.sql

```diff
@@ -0,0 +1,18 @@
+create table work_offer (
+    id uuid,
```

Comment:
Can we change this type (and for other ids) to `bigserial`, so we will be consistent with the `city` table?

Reply:
I wanted to make the offer ids un-guessable. With autogenerated numerical ids that's hard to enforce. Though maybe it's not a problem, cause the offers are supposed to be public.

Reply:
Good idea! We should follow this pattern then.